client/state/ui/reader/fullpost/reducer.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_FULLPOST_SET_VISIBILITY,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+
+export function isVisible( state = false, action ) {
+	switch ( action.type ) {
+		case READER_FULLPOST_SET_VISIBILITY:
+			return action.show;
+		case SERIALIZE:
+			return false;
```

Comment:
@samuelclemens @blowery :+1: we're planning on not persisting anything in the UI tree. #3337